### PR TITLE
App: Highlight search results

### PR DESCRIPTION
**Pull request by TommyBom**

### CODE REVIEW

File: src/app/features/ahbs/components/ahb-table/ahb-table.component.spec.ts [hunk 1]
```diff
@@ -1,22 +1,13 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { AhbTableComponent } from './ahb-table.component';
+import { MockBuilder, MockRender, ngMocks } from 'ng-mocks';
```
Comment (hf-krechan) [inline src/app/features/ahbs/components/ahb-table/ahb-table.component.spec.ts:2]: I know mocks only from unit tests. But these lines look like production code.
What is the purpose of the MockBuilder here?

File: src/app/features/ahbs/views/ahb-page/ahb-page.component.spec.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ describe('AhbPageComponent', () => {
       pruefi: '123',
     });
     const html = ngMocks.formatHtml(fixture);
-    expect(html).toContain('<app-header>');
+    expect(html).toContain('<app-header');
```
Comment (hf-krechan) [inline src/app/features/ahbs/views/ahb-page/ahb-page.component.spec.ts:27]: What will happen is the string is not contained in the html?

File: src/app/shared/pipes/highlight.pipe.spec.ts
Comment (hf-krechan) [inline src/app/shared/pipes/highlight.pipe.spec.ts:1]: this file seems to be test.
In Python the test files must start or end with `test`.
How does the test framework find all tests?

**Review by hf-krechan — APPROVED** [review]
looks good for a frontend noobie ;)

File: src/app/features/ahbs/components/ahb-table/ahb-table.component.spec.ts (hunk 1 above)
Comment (TommyBom) [reply src/app/features/ahbs/components/ahb-table/ahb-table.component.spec.ts:2]: files that end in `.spec.ts` are unit tests.

File: src/app/features/ahbs/views/ahb-page/ahb-page.component.spec.ts (hunk 2 above)
Comment (TommyBom) [reply src/app/features/ahbs/views/ahb-page/ahb-page.component.spec.ts:27]: This unit test will fail. This is just a simple example test and could be expanded with more detailed tests

File: src/app/shared/pipes/highlight.pipe.spec.ts
Comment (TommyBom) [reply src/app/shared/pipes/highlight.pipe.spec.ts:1]: The end with `spec.ts`

**Pull request merged**